data/json/recipes/weapon/mods.json

```diff
@@ -65,5 +65,96 @@
       [ [ "scrap", 1 ] ],
       [ [ "plastic_chunk", 1 ] ]
     ]
+  },
+  {
+    "type": "recipe",
+    "result": "red_dot_sight",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_MODS",
+    "skill_used": "electronics",
+    "difficulty": 5,
+    "skills_required": [ "gun", 3 ],
+    "time": 15000,
+    "decomp_learn": 2,
+    "book_learn": [ [ "manual_electronics", 2 ], [ "textbook_electronics", 2 ] ],
+    "using": [ [ "soldering_standard", 10 ], [ "surface_heat", 10 ] ],
+    "qualities": [ { "id": "SCREW_FINE", "level": 1 } ],
+    "tools": [ [ [ "mold_plastic", -1 ] ] ],
+    "components": [ 
+        [ [ "plastic_chunk", 2 ] ],
+        [ [ "amplifier", 1 ] ],
+        [ [ "power_supply", 1 ] ],
+        [ [ "battery", 100 ], [ "plut_cell", 1 ] ],
+        [ [ "lens", 2 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "tele_sight",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_MODS",
+    "skill_used": "fabrication",
+    "difficulty": 5,
+    "skills_required": [ [ "gun", 4 ] ],
+    "time": 15000,
+    "decomp_learn": 3,
+    "book_learn": [ [ "manual_rifle", 2 ], [ "manual_fabrication", 3 ], [ "textbook_fabrication", 3 ] ],
+    "using": [ [ "soldering_standard", 10 ], [ "surface_heat", 10 ] ],
+    "qualities": [ { "id": "SCREW_FINE", "level": 1 } ],
+    "tools": [ [ [ "mold_plastic", -1 ] ] ],
+    "components": [ 
+        [ 
+            [ "scrap", 1 ], [ "can_drink_unsealed", 1 ], [ "can_food_unsealed", 1 ],
+            [ "canister_empty", 1 ], [ "plastic_chunk", 1 ]
+        ],
+        [ [ "lens", 2 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "pistol_scope",
```

Comment:
You made a new "improvised scope" and made it craftable, but you also made the pistol scope, which is much, much better also craftable.
Either make another improvised scope for pistols, or just add the tele_sight.